nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -226,13 +226,22 @@ def _add_weight_quantization_target_point(self, quantization_point: SingleConfig
         node_name = quantization_point.insertion_point.target_node_name
         node = nncf_graph.get_node_by_name(node_name)
         port_id = self._backend_entity.get_weight_tensor_port_id(model, node)
-        if port_id is None:
-            raise RuntimeError(f'Could not find the port_id for the node {node_name}')
         weight_quantization_target_point = self._backend_entity.target_point(TargetType.OPERATION_WITH_WEIGHTS,
                                                                              node_name,
                                                                              port_id)
         self._quantization_target_points.add(weight_quantization_target_point)
 
+    def _get_activation_quantization_axis(self, quantizer_config: QuantizerConfig) -> Optional[int]:
+        """
+        Returns quantization axis for activation quantizer.
+
+        :param quantizer_config: Quantization configuration
+        :return: Quantization axis in per-channel case. None in per-tensor case.
+        """
+        if quantizer_config.per_channel:
+            return 1
+        return None
```

Comment:
I think it should be backend specific function because OpenVINO and TF support NHWC layout.
@KodiaqQ, @andrey-churkin please correct me if I am wrong.

Reply:
Agree, we should move it into the backend entity.

Reply:
Done